KSCrash.podspec

```diff
@@ -16,123 +16,54 @@ Pod::Spec.new do |s|
   s.xcconfig = { 'GCC_ENABLE_CPP_EXCEPTIONS' => 'YES' }
   s.default_subspecs = 'Installations'
 
-  s.subspec 'Recording' do |recording|
-    recording.compiler_flags = '-fno-optimize-sibling-calls'
```

Comment:
Why is this `-fno-optimize-sibling-calls` flag removed?

Reply:
I guess we can set it in CocoaPods, but in SPM it can only be set with `.unsafeFlags`, which is not allowed in "release" (specified by version), but is allowed in development (local package or specified by commit or branch).

Reply:
It's also unclear which functions this flag is for, so we can't know if it's needed in `Recording` or in `RecordingCore`.

Reply:
Well, you've added it here #379 :) 
Looks like it's actually needed. But maybe we can take a solution from https://github.com/rollbar/rollbar-apple/pull/335 (if I understand that fix correctly).